docs/configuration.md

```diff
@@ -10,6 +10,12 @@ For more information on how the configuration is being loaded and used, please r
 ### Application Config
 Certain features of the React application import variables from an [AppConfig](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/config/config.ts#L5) object. The configuration can be customized by modifying [config-custom.ts](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/config/config-custom.ts).
 
+#### Example: Add a custom logo
+
+1. Add your logo to the folder in `amundsen_application/static/images/`
+2. Set the the `logoPath` key on the `AppConfigCustom` object in [config-custom.ts](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/config/config-custom.ts). For example Lyft uses the value `"/static/images/lyft-logo.svg"`
+3. Rebuild/redeploy. The `npm run build` step will rebuild scripts file to reference the new logo image, and the `python3 setup.py install` step will updated deployed files and make the new image available on the server.
```

Comment:
I know I originally wrote this in slack, but I wonder if splitting step 3 into two steps (rebuild and deploy) makes more sense. I think it works either way, so I'll leave it to you.

Reply:
Well, you contributed first. I just carried it a few steps 😉 

I was thinking the same with a bit more splitting. Anyway, important part is content is there... we can keep bikeshedding forever.

 Please merge at will!